libs/hooks/src/use-notifications.ts

```diff
@@ -1,45 +1,44 @@
-import { useEffect, useRef, useState } from 'react';
+import { useState } from 'react';
 import getSDK from '@akashaorg/core-sdk';
-import { GQL_EVENTS } from '@akashaorg/typings/lib/sdk';
 
 /**
- * Hook to listen for mutation events emitted from the SDK's globalChannel.
- * @example useListenForMutationEvents hook
+ * Hook to enable notifications and observe signature propmpt result.
+ * @example useNotifications hook
  * ```typescript
- * const { messageObj, appid, success, pending } = useListenForMutationEvents();
+ * const { notificationsEnabled, previouslyEnabled, waitingForSignature, enableNotifications } = useNotifications();
  *
  * ```
  */
-export function useListenForMutationEvents() {
-  const [data, setData] = useState(null);
-
+export function useNotifications() {
   const sdk = getSDK();
 
-  const messageObj = useRef(null);
-  const uuid = useRef('');
+  const [previouslyEnabled, setPreviouslyEnabled] = useState(() =>
+    sdk.services.common.notification.getNotificationsEnabledStatus(),
```

Comment:
`getNotificationsEnabledStatus` can throw an error as it performs parsing of status and it need to be handled.

Reply:
done 👍